packages/mobile/src/redux/store.test.ts

```diff
@@ -137,6 +137,7 @@ describe('store state', () => {
           "celoEducationUri": null,
           "celoEuroEnabled": true,
           "dappListApiUrl": null,
+          "dappSessionActive": null,
```

Comment:
The schema version should be bumped as best practice.

Reply:
😅 updated